File: LeetCode/python/runningSum.py
class Solution(object):
    def runningSum(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        running_sum = []
        for i in range(len(nums)):
            running_sum.append(sum(nums[:i+1]))
        return running_sum

File: LeetCode/typescript/runningSum.ts
function runningSum(nums: number[]): number[] {
  let runningSum: any[] = [];
  let sum: number = 0;
  for (let i = 0; i < nums.length; i++) {
    sum = sum + nums[i];
    runningSum.push(sum);
  }
  return runningSum;
}
